tools/workspace-plugin/src/generators/react-component/files/component/__componentName__.tsx__tmpl__

```diff
@@ -12,6 +13,9 @@ export const <%= componentName %>: ForwardRefComponent<<%= componentName %>Props
   const state = use<%= componentName %>_unstable(props, ref);
 
   use<%= componentName %>Styles_unstable(state);
+  use<%= componentName %>Styles_unstable(state);
+  // TODO update types in packages/react-components/react-shared-contexts/src/CustomStyleHooksContext/CustomStyleHooksContext.ts
```

Comment:
can we provide more docs on what is this todo about ? why would I need to update types in completely different package ?

Reply:
What form would you prefer a longer explanation, would it be OK to just have more inline comments?

Reply:
can it point to some md file that provides doc  for it ?

Reply:
Added a link to the RFC that proposed the custom styling hook solution https://github.com/microsoft/fluentui/blob/master/rfcs/react-components/convergence/custom-styling.md